idea_town/frontend/static-src/app/views/experiment-page.js

```diff
@@ -42,13 +42,16 @@ export default PageView.extend({
       hook: 'feedback'
     }],
 
-    'model.modified_date': {
-      type: (el, value) => new Date(value),
+    'model.modified': {
+      type: (el, value) => {
+        const d = new Date(value);
+        el.innerHTML = `${d.getMonth() + 1} / ${d.getDate()} / ${d.getFullYear()}`;
```

Comment:
Not sure if we want to use [`toLocaleDateString()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString) or something similar here to avoid a potential "MM/DD/YYYY" vs "DD/MM/YYYY" fist fight.